File: Java/백준/Bronze/29863. Arno's Sleep Schedule/Arno's Sleep Schedule.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int sleep = sc.nextInt();
        int wake = sc.nextInt();
        
        int total = 0;
        
        if (sleep >= 20) {
            total = (24 - sleep) + wake;
        } else {
            total = wake - sleep;
        }
        
        System.out.print(total);
        
        sc.close();
    }
}

File: C++17/백준/Bronze/29863. Arno's Sleep Schedule/Arno's Sleep Schedule.cc
#include <iostream>
using namespace std;

int main (void) {
    int sleep, wake;
    cin >> sleep >> wake;
    
    int total = 0;
    
    if (sleep >= 20) {
        total = (24 - sleep) + wake;
    } else {
        total = wake - sleep;
    }
    
    cout << total;
    
    return 0;
}